docs/extras/dump-games.md

```diff
@@ -21,26 +21,29 @@ If you intend to use this guide to share your dumped games, don't.
 
 - Your SD Card needs to have enough space to fit the game you want to dump.
 - The latest release of [WUP Installer GX2](https://wiiubru.com/appstore/zips/wup_installer_gx2.zip).
-- The [disc2app](https://wiiubru.com/appstore/zips/disc2app.zip) homebrew application.
+- The [wudd](https://github.com/wiiu-env/wudd/releases) homebrew application.
 
 ### Instructions {docsify-ignore}
 
 1. Copy the contents of the `wup_installer_gx2.zip` file to the root of your SD Card.
-1. Copy the contents of the `disc2app.zip` file to the root of your SD Card.
+1. Copy the contents of the newly downloaded wudd `.zip` file to the root of your SD Card.
 1. Take the SD Card out of your PC and plug it into your Wii U.
 1. Power on your Wii U.
-1. Eject your game disc if you have one inserted.
-1. Launch the Homebrew Launcher and start the disc2app app.
-1. Press the A button to start the dumping process to the SD Card.
-1. When prompted, insert the game disc you want to dump.
-1. When the process is complete, your Wii U will launch the Wii U Menu.
-1. Launch the Homebrew Launcher and start the WUP Installer GX2 app.
-1. Select your game (`WUP-X-XXXX`), press `Install` and confirm with `Yes`.
+1. Launch the Homebrew Launcher and start the wudd app.
+1. Change dump location to the SD Card.
+1. Select `Dump partition as .app`
+1. Select the `Game` partition to start dumping.
+1. When finished, exit wudd and navigate back to the Wii U Menu.
+1. Insert your SD Card into your computer.
+1. Copy the `GMXXXXXXXXXXXXXXXX` folder from `sd:/wudump/WUP-X-XXXX` to the `install` folder on your SD Card.
+1. Insert the SD Card.
```

Comment:
```suggestion
1. Insert the SD Card into your Wii U.
```

Reply:
Yeah I noticed it after pushing but since of temperature reasons I wasn't able to fix it until now.